### PR TITLE
Added the MonthlySavingsPlanAmortizaton, PremiumSupportFee

### PR DESCRIPTION
**Pull request by athiruma**
1. Add the payer account monthly Saving Plan Amortization, and Premium Support Fee to the cost payer reports.
2. It takes the monthly savings plan input from the GSheet ASP.

### CODE REVIEW

File: cloud_governance/policy/aws/cost_explorer_payer_billings.py [hunk 1]
```diff
@@ -11,7 +13,11 @@
...
 
 class CostExplorerPayerBillings(CostBillingReports):
-    """This class is responsible for generation cost billing report for Budget, Actual, Forecast from the Org level"""
+    """
+    This class is responsible for generation cost billing report for Budget, Actual, Forecast from the Org level
+    Monthly savings Plan Amortization: (linked_account_total_cost/payer_account_total_cost) * monthly_savings_plan_cost
+    Monthly_support_fee: (monthly_support_fee - (monthly_support_fee * discount ) ) * (linked_account_total_cost/payer_account_total_cost)
+    """
 
     TEMP_DIR = '/tmp'
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer_payer_billings.py:22]: Can you fetch TEMP_DIR from env. variable ?

Comment (athiruma) [reply cloud_governance/policy/aws/cost_explorer_payer_billings.py]: done

File: cloud_governance/policy/aws/cost_explorer_payer_billings.py [hunk 2]
```diff
@@ -22,6 +26,10 @@ def __init__(self):
         self.__ce_client = boto3.client('ce', aws_access_key_id=self.__access_key, aws_secret_access_key=self.__secret_key, aws_session_token=self.__session)
         self.__cost_explorer_operations = CostExplorerOperations(ce_client=self.__ce_client)
         self.__replacement_account = literal_eval(self._environment_variables_dict.get('REPLACE_ACCOUNT_NAME'))
+        self.__savings_discounts = float(self._environment_variables_dict.get('PAYER_SUPPORT_FEE_CREDIT', 0))
+        self.__monthly_cost_for_spa_calc = {}
+        self.__monthly_cost_for_support_fee = {}
+        self.__temporary_dir = self._environment_variables_dict.get('TEMPORARY_DIR', '/tmp')
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer_payer_billings.py:32]: I think you should add it to [environment_variables]( https://github.com/redhat-performance/cloud-governance/blob/main/cloud_governance/main/environment_variables.py)

Comment (athiruma) [reply cloud_governance/policy/aws/cost_explorer_payer_billings.py]: Already added it to environment_variables, but added it here too.
Now removed.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**